sdk/clientcore/annotation-processor/ci.yml

```diff
@@ -39,6 +39,7 @@ extends:
       - name: annotation-processor
         groupId: io.clientcore
         safeName: annotationprocessor
+        skipPublishDocMs: true
```

Comment:
Do we have an issue reported to MS docs to support publishing docs with `io.clientcore`?

Reply:
@srnagar - You were on that email (subject: RE: Documenting Java packages/modules), there is an issue logged on the docs side and this is also something I bring up every week in my docs meeting. Trust me, this is NOT going to fall through the cracks but the downside is that things aren't done in a timely manner on their side.